__tests__/api-paths.test.js

```diff
@@ -14,15 +14,26 @@ describe("GET /api/categories", () => {
       .expect(200)
       .then((data) => {
         const result = data.body.categories;
-        expect(result[0]).toEqual(
-          expect.objectContaining({
-            slug: expect.any(String),
-            description: expect.any(String),
-          })
+        expect(result).toEqual(
+          expect.objectContaining([
```

Comment:
Ideally you want a forEach, or some kind of loop, then you want to check each object inside of there. 
At the moment, this checks there's an array with at least one object that looks like this. 
We want to make sure all the objects look like this.